### ISSUE TITLE
Control.toggle function

### ISSUE THREAD

**Issue by brionco** [bitfocus/companion-module-qsys-remote-control#10]
Control.toggle only seems to works if you have feedback setup for the same button.

Only an issue if you want to have a blind toggle, or if you want the feedback from another source.

**Comment by phillipivan** [comment]
This is correct. I have added a `static-text` warning to this action to inform users of this.

**Comment by phillipivan** [comment]
Fixed in 3.0.0
